pytorch_lightning/trainer/trainer.py

```diff
@@ -1740,10 +1742,6 @@ def checkpoint_callbacks(self) -> List[ModelCheckpoint]:
         in the Trainer.callbacks list."""
         return [c for c in self.callbacks if isinstance(c, ModelCheckpoint)]
 
-    @property
-    def resume_from_checkpoint(self) -> Optional[Union[str, Path]]:
-        return self.checkpoint_connector.resume_checkpoint_path
```

Comment:
we can't delete this outright. this is a public property, part of the trainer API, which needs to go through the deprecation cycle before its removal